apps/meow/src/meow.ts

```diff
@@ -181,9 +181,12 @@ export class MEOW {
     await this.store.init();
   }
 
-  async sendMeow (msg: string): Promise<void> {
+  async sendMeow (msg: string, publishOnTwitter: boolean = true): Promise<void> {
```

Comment:
publishontwitter flag should default to false i think . so it works for previous sendMeow without twitter flag already in code

Reply:
It's actually the other way round. If a user has enabled twitter (`meow.enableTwitter()`), then it would by default publish all meows to twitter. You can pass `false` to override this. eg. `meow.sendMeow("hello..", false);`